Makefile

```diff
@@ -275,7 +271,7 @@ gcloud-%: gcloud
 node-%: node
 	@ echo "# Installing $*..."
 	# Hack to (more quickly) detect whether a package is already installed (available in node).
-	cd webapp; node -p "require('$*/package.json').version" 2>/dev/null || npm install --no-save $*
+	cd webapp; node -p "require('$*epackage.json').version" 2>/dev/null || npm install --no-save $*
```

Comment:
What is this change about?

Reply:
That must be some devil living in my vim :)

```suggestion
	cd webapp; node -p "require('$*/package.json').version" 2>/dev/null || npm install --no-save $*
```

Reply:
That must be some devil living in my vim :)

```suggestion
	cd webapp; node -p "require('$*/package.json').version" 2>/dev/null || npm install --no-save $*
```